apps/web-giddh/src/app/userDetails/components/subscriptions/subscriptions.component.scss

```diff
@@ -93,7 +93,7 @@
 }
 .subscription-dropdown li:hover a,
 .subscription-dropdown li:hover {
-    background: $white-smoke;
```

Comment:
@aditi-tiwari17 @shalinee-walkover Please check this code revert

Reply:
checked, its fine